docs/source/usage.rst

```diff
@@ -123,6 +123,11 @@ in scan coordinates on the right.
 
 ..  figure:: ./images/use/apply.png
 
+Instead of average, you can select "Standard Deviation". This calculates
+standard deviation of all detector frame.
```

Comment:
Better: "all detector frame**s**". Maybe also some better wording can be found to describe over which axes the reduction is done, but maybe the screenshot is enough to intuitively understand this.

In my opinion, the screenshot should should have stddev already enabled and show the result, in addition of showing the drop-down. Similar to the rect screenshot already showing the rectangular ROI.